docker/cygnus-twitter/Dockerfile

```diff
@@ -43,18 +44,19 @@ ENV FLUME_TGZ "apache-flume-${FLUME_VER}-bin.tar.gz"
 ENV FLUME_URL "http://archive.apache.org/dist/flume/${FLUME_VER}/${FLUME_TGZ}"
 ENV FLUME_HOME "/opt/apache-flume"
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.6.0
+ENV JAVA_VERSION=1.7.0
```

Comment:
Is this really working, by using the `=`? It seems the environment variables in docker have the form `ENV name value`, without the equals character.